src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c

```diff
@@ -254,6 +254,22 @@ static int statx_wrapper(int dirfd, const char *restrict pathname, int flags,
 }
 #endif
 
+#if defined(__linux__) && defined(__arm__)
+/**
+ * Lookup functions with time_t parameter. Try to use 64 bit symbol
+ * if sizeof(time_t) exceeds 32 bit.
+ */
+static void* lookup_time_t_function(const char* symbol, const char* symbol64) {
+    void *func_ptr = NULL;
+    if (sizeof(time_t) > 4) {
+        func_ptr =  dlsym(RTLD_DEFAULT, symbol64);
+    }
+    if (func_ptr == NULL)
```

Comment:
It might be more readable if both `if` blocks had either no brackets or both had brackets.